Anlab.Jobs.MoneyMovement/Program.cs

```diff
@@ -0,0 +1,137 @@
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json");
+
+
+            var environmentName = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+
+            if (string.Equals(environmentName, "development", StringComparison.OrdinalIgnoreCase))
+            {
+                builder.AddUserSecrets<Program>();
+            }
+
+
+            builder.AddEnvironmentVariables();
+            Configuration = builder.Build();
+            
+
+            IServiceCollection services = new ServiceCollection();
+            services.AddDbContext<ApplicationDbContext>(options =>
+                    options.UseSqlite("Data Source=..\\..\\..\\..\\Anlab.Mvc\\bin\\Debug\\netcoreapp1.1\\anlab.db")
+                // options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"))
+            );
+            Provider = services.BuildServiceProvider();
+
+            Console.WriteLine("About to start");
+            var result = Task.Run(() => ProcessOrders()).Result; //Wasn't able to debug this unless it returned a result...
+
+
+
+            Console.WriteLine("Done");
+        }
+
+        public static async Task<bool> ProcessOrders()
+        {
+            var dbContext = Provider.GetService<ApplicationDbContext>();
+            var token = Configuration.GetSection("MoneyMovement:SlothApiKey").Value;
+            var url = Configuration.GetSection("MoneyMovement:SlothApiUrl").Value;
+
+
+            using (var client = new HttpClient())
+            {
+                var orders = dbContext.Orders.Include(i => i.ApprovedPayment).Where(a =>
+                    a.ApprovedPayment != null && a.Paid && a.Status != OrderStatusCodes.Complete).ToList();
+                client.BaseAddress = new Uri(url);
+                client.DefaultRequestHeaders.Add("X-Auth-Token", token);
+
+                if (orders?.Count > 0)
+                {
+                    Console.WriteLine($"Processing {orders.Count} orders");
+                    var updatedCount = 0;
+                    foreach (var order in orders)
+                    {
+                        var response = await client.GetAsync(order.ApprovedPayment.Transaction_Id);
+                        if (response.StatusCode == HttpStatusCode.NotFound)
+                        {
+                            continue;
+                        }
+                        if (response.StatusCode == HttpStatusCode.NoContent)
+                        {
+                            continue;
+                        }
+                        if (response.IsSuccessStatusCode)
+                        { 
+                            var slothResponse = await response.GetContentOrNullAsync<Sloth>();
+                            slothResponse.JsonDetails = await response.Content.ReadAsStringAsync(); //TODO: Decide if we really need this.

```

Comment:
@jSylvestre Yea I don't think we need to care about any of this, maybe we'll store the KFS Key but we don't particularly need that either since we can just look it up based on transaction_id.